docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc

```diff
@@ -151,6 +151,24 @@ The selection can be changed at any time in the _Global_ settings for _Role_.
 image::monitoring-console/mc_setting_global_rolesync.png[Role Setting]
 
 
+[[monitoring-console-page-settings]]
+==== Page Settings
+The individual page settings are explained in more detail in the context of the operations they are involved in. This table gives an additional overview.
+
+.Page Settings
+[cols="1,1,4",options="header"]
+|====================
+| Property | Default | Description
+| Name | - | Name of the page; only editable for pages that do not originate from a preset
+| Page Rotation | yes | Whether or not to include the page in the page rotation
+| Type | _Manual_ | _Manual_ page (widgets added by the user) or _Query_ page (widgets added from query matches)
+| Add Widgets (_Manual_ only) | - |   Selection of the series to add as a widget to the current page
+| Max Size (_Query_ only) | - | Maximum number of widgets to place on the page from query matches
+| Query Series (_Query_ only) | - | The series pattern to use to query for series to place on the page
+| Query Interval (_Query_ only) | - | Number of seconds between repopulating the page with widgets from query matches, can be given using any time unit, e.g. `1min`, `30sec`, `1h`
+| Sync (non _Guest_ role only) | _Push_ to update the shared state with the local page, _Pull_ to update the local state with the shared state
+| Sync (_Admin_ role only) auto | yes | Whether or not to push changes to the local page to the shared state automatically when using the admin role
+
```

Comment:
The table with Page settings is broken, can you please review it and fix, @jbee? Look at this: https://github.com/payara/Payara-Community-Documentation/blob/ebd66b82a8fd910a7dc3511bbc598a3c1faca5a1/docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc#page-settings

For example, this one fixes it (yes as the default option for non-guest, table ending at the end):

```suggestion
| Sync (non _Guest_ role only) | yes | _Push_ to update the shared state with the local page, _Pull_ to update the local state with the shared state
| Sync (_Admin_ role only) auto | yes | Whether or not to push changes to the local page to the shared state automatically when using the admin role
|====================
```